games-fps/chocolate-doom/chocolate_doom-3.0.0.recipe

```diff
@@ -0,0 +1,128 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="https://www.chocolate-doom.org/"
+COPYRIGHT="1993-1996, Id Software, Inc
+	1993-2008, Raven Software
+	1996, Rogue Entertainment / Velocity, Inc.
+	1996-2013, Free Software Foundation, Inc
```

Comment:
You forgot the dot after "Inc" at lines 6 and 9.

BTW, we also prefer to drop the commas after the copyright years:
```diff
-COPYRIGHT="1993-1996, Id Software, Inc
+COPYRIGHT="1993-1996 Id Software, Inc.
-	1993-2008, Raven Software
+	1993-2008 Raven Software
-	1996, Rogue Entertainment / Velocity, Inc.
+	1996 Rogue Entertainment / Velocity, Inc.
-	1996-2013, Free Software Foundation, Inc
+	1996-2013 Free Software Foundation, Inc.
-	1999, Thomas A. Fine
+	1999 Thomas A. Fine
-	2002-2003, Marcel Telka
+	2002-2003 Marcel Telka
-	2002-2010, The DOSBox Team
+	2002-2010 The DOSBox Team
-	2005-2014, Simon Howard
+	2005-2014 Simon Howard
-	2005-2014, Andrey Budko
+	2005-2014 Andrey Budko
-	2006, Ben Ryves
+	2006 Ben Ryves
-	2008, GhostlyDeath
+	2008 GhostlyDeath
-	2008, David Flater
+	2008 David Flater
-	2010, James Haley, Samuel Villarreal
+	2010 James Haley, Samuel Villarreal
-	2011, Team Eternity
+	2011 Team Eternity
-	2014, Fabian Greffrath"
+	2014 Fabian Greffrath"
```

Reply:
What's the status on this one?